src/lib/Controls.js

```diff
@@ -183,40 +183,46 @@ class Controls {
     };
 
     /**
-     * Adds buttons to controls
+     * Adds element to controls
      *
      * @public
-     * @param {string} text - button text
-     * @param {Function} handler - button handler
+     * @param {string} text - text
+     * @param {Function} handler - on click handler
      * @param {string} [classList] - optional class list
-     * @param {string} [buttonContent] - Optional button content HTML
-     * @return {void}
+     * @param {string} [content] - Optional content HTML
+     * @param {string} [tag] - Optional html tag, defaults to 'button'
+     * @return {HTMLElement} The created HTMLElement inserted into the control
      */
-    add(text, handler, classList = '', buttonContent = '') {
+    add(text, handler, classList = '', content = '', tag = 'button') {
         const cell = document.createElement('div');
         cell.className = 'bp-controls-cell';
 
-        const button = document.createElement('button');
-        button.setAttribute('aria-label', text);
-        button.setAttribute('type', 'button');
-        button.setAttribute('title', text);
-        button.className = `${CONTROLS_BUTTON_CLASS} ${classList}`;
-        button.addEventListener('click', handler);
+        const element = document.createElement(tag);
+        element.setAttribute('aria-label', text);
+        element.setAttribute('title', text);
+
+        if (tag === 'button') {
+            element.setAttribute('type', 'button');
```

Comment:
Create const for 'button' since being used multiple times?

Reply:
They're the same string, but they represent different things, tag name vs button type -- I fee like a constant here would be convenient but possibly misleading. Thoughts?

Reply:
I agree with @ConradJChan. Further, we should only advocate for constants where they add clarity rather than as an ironclad requirement.